newsloom/streams/services/_google.py

```diff
@@ -23,6 +23,7 @@ def search_google(
     max_results_per_keyword: int = 5,
     days_ago: int | None = None,
     search_type: str = "news",
+    **_kwargs
```

Comment:
Added **_kwargs parameter to search_google function to properly handle additional parameters passed when running in debug mode.